Hakuba/Source/Common/Utilities.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Utilities.swift
+//  Example
+//
+//  Created by Keeeeen on 2018/08/09.
+//
+
+import Foundation
+
+func classNameOf(_ aClass: AnyClass) -> String {
+    return String(describing: aClass).components(separatedBy: ".").last!
```

Comment:
Can we remove `.components(separatedBy: ".").last!` ?

Reply:
That's fine. I tried and tested